src/scratchtocatrobat/converter/converter.py

```diff
@@ -871,13 +871,28 @@ def _catrobat_sound_from(scratch_sound):
         sound_resource_name = scratch_sound[scratchkeys.SOUND_NAME]
         soundinfo.setSoundFileName(mediaconverter.catrobat_resource_file_name_for(sound_md5_filename, sound_resource_name))
         return soundinfo
+    @staticmethod
+    def _variable_exists(var, catrobat_project):
+        return var in [check_var.getName() for check_var in catrobat_project.getProjectVariables()]
+    
+    @staticmethod
+    def _create_show_text_brick_and_update_positions(var_object, context):
+        show_variable_brick = catbricks.ShowTextBrick(context.visible_var_X, context.visible_var_Y)
+        show_variable_brick.setUserVariableName(var_object.getName())
+        show_variable_brick.setUserVariable(var_object)
+        context.visible_var_Y -= VISIBLE_VAR_POSITION_STEP_Y
+        if context.visible_var_Y <= VISIBLE_VAR_POSITION_THRESHOLD_Y:
+            context.visible_var_Y = VISIBLE_VAR_Y_INIT
+            context.visible_var_X += VISIBLE_VAR_POSITION_STEP_X
+        if context.visible_var_X >= VISIBLE_VAR_POSITION_THRESHOLD_X:
+                log.info("Too many visible variables")
```

Comment:
The log can't be seen by users on their mobile devices. You could possibly additionally add a Note Brick with an appropriate warning to the script.